src/main/java/ru/buyanov/hunting/IntIterable.java

```diff
@@ -17,16 +18,23 @@ public IntIterable(int[] backed) {
         return new IntIterator();
     }
 
+    /**
+     * Weakly consistent iterator for {@link IntIterable}
```

Comment:
Fail-fast Iterator implementation is not acceptable here, because IntIterable class allows package-wide direct access to array.
The task did not infer modification of outer class, so it is considered forbidden for modification.